app/assets/stylesheets/application.scss

```diff
@@ -113,17 +113,25 @@ h5 {
 }
 
 .div-bar {
-  background-color: #37becc;
+  background-color: $primary-color;
   color: #ffffff;
   padding: 10px;
   margin: 0px;
   margin-top: 15px;
+  box-shadow: 5px 5px 5px #b1b1b1;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+}
+
+.div-bar a {
+  color: $secondary-color;
 }
 
 .content {
   background-color: #fff;
-  margin: 0;
-  padding: 7px;
+  margin: 0 0 30px 0;
+  padding: 0;
+  box-shadow: 5px 5px 5px #b1b1b1;
```

Comment:
Color literals like `#b1b1b1` should only be used in variable declarations; they should be referred to via variable everywhere else.